app/controllers/internal/configs_controller.rb

```diff
@@ -22,8 +24,16 @@ def config_params
       ga_view_id ga_fetch_rate
       mailchimp_newsletter_id mailchimp_sustaining_members_id
       mailchimp_tag_moderators_id mailchimp_community_moderators_id
-      periodic_email_digest_max periodic_email_digest_min
+      periodic_email_digest_max periodic_email_digest_min suggested_tags
     ]
     params.require(:site_config).permit(allowed_params)
   end
+
+  def clean_up_params
+    params[:site_config][:suggested_tags] = params[:site_config][:suggested_tags].delete(" ") if params[:site_config][:suggested_tags]
```

Comment:
I think this should also lowercase all the tags, because right now I can enter `Beginners` and IIRC we have had issues with casing

Reply:
Yeah that makes sense.